src/commands/appSettings/filterDownloadAppSettings.ts

```diff
@@ -11,42 +11,75 @@ import { localize } from "../../localize";
 export async function filterDownloadAppSettings(context: IActionContext, sourceSettings: { [key: string]: string }, destinationSettings: { [key: string]: string }, destinationName: string): Promise<void> {
     let suppressPrompt: boolean = false;
     let overwriteSetting: boolean = false;
+    let suppressPromptOfSettingsToIgnore = false;
+    let overwriteSettingsToIgnore: boolean = false;
 
     const addedKeys: string[] = [];
     const updatedKeys: string[] = [];
     const userIgnoredKeys: string[] = [];
     const matchingKeys: string[] = [];
+    const securitySettingsIgnored: string[] = [];
+    const listOfSettingsToIgnore: string[] = ["AzureWebJobsStorage", "WEBSITE_CONTENTAZUREFILECONNECTIONSTRING", "WEBSITE_CONTENTSHARE"];
 
     for (const key of Object.keys(sourceSettings)) {
-        if (destinationSettings[key] === undefined) {
-            addedKeys.push(key);
-            destinationSettings[key] = sourceSettings[key];
-        } else if (destinationSettings[key] === sourceSettings[key]) {
-            matchingKeys.push(key);
-        } else if (sourceSettings[key]) { // ignore empty settings
-            if (!suppressPrompt) {
+        if (listOfSettingsToIgnore.includes(key)) {
```

Comment:
does this check need to be case insensitive? Would the appsettings set in all lowercase or weird casing still work in the runtime?

Reply:
listOfSettingsToIgnore is an array that I manually created which contains the names of the three unsafe app settings we decided we would should prompt the user if they would like to ignore. I don't think the user would be able to modify that since it would be a list pre-determined by us but I could still add a check just in case.

Reply:
but the user can update their appsettings anytime they like.. I think in this case.. IF you have access to those app settings on the portal.. change their casing, see if the runtime still operates? if so, your checks should be case insensitive.

Reply:
that makes sense. I'll make it so that the check is case insensitive.